fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/derby/DerbyFhirDatabase.java

```diff
@@ -108,21 +126,18 @@ public void populateResourceTypeAndParameterNameTableEntries() throws SQLExcepti
         try (Connection connection = getConnection()) {
 
             // Ensures we don't double up the generated derby db prepopulation.
-            // Docs for the table are at https://db.apache.org/derby/docs/10.5/ref/rrefsistabs24269.html
-            boolean process = true;
-            final String sql = "SELECT COUNT(TABLENAME) AS CNT FROM SYS.SYSTABLES WHERE TABLENAME = 'PARAMETER_NAMES'";
-            try (PreparedStatement stmt = connection.prepareStatement(sql)) {
-                stmt.execute();
-                ResultSet set = stmt.getResultSet();
-                if (set.next()) {
-                    int val = set.getInt("CNT");
-                    if (val > 0) {
-                        process = false;
-                    }
+            boolean populated;
+            final String sql = "SELECT 1 FROM PARAMETER_NAMES FETCH FIRST 1 ROWS ONLY";
```

Comment:
Is this too optimistic? What happens if the table isn't deployed?

Reply:
Why would the table not be deployed? This is called after the schema has been generated, so if the table doesn't exist, we have bigger issues.

Reply:
I am pretty sure that's why I wrote the sys.systables query as we were hitting that occasionally when the Schema failed.